_data/pprathameshmore/pprathameshmore.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Prathamesh More # No longer than 18 characters
+institution: Zeal College of Engineering and Research 🚩 # no longer than 58 characters
+profile_pic: pprathameshmore.jpg # Name and extension of your profile picture(ex. mona.png)
```

Comment:
there is an issue here with the image filename, please look into it when you open an other PR